modules/orionode.debug.server/adapters.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
I put four adapters here as the "default" setup, because configuring an adapter is complicated. Adapters, especially the community ones, have different build command and many of them relies on VS Code specific functionalities such as executing extension commands. As a result, the user has to dig into the source code and find the right way to set up the adapter, which cost plenty of time.
However, I don't think we should be responsible for maintaining this configuration for community adapters.
Any suggestion?